src/PowerShellEditorServices/Services/TextDocument/Handlers/CompletionHandler.cs

```diff
@@ -202,62 +202,65 @@ public void SetCapability(CompletionCapability capability)
             string completionText = completionDetails.CompletionText;
             InsertTextFormat insertTextFormat = InsertTextFormat.PlainText;
 
-            if ((completionDetails.CompletionType == CompletionType.Variable) ||
-                (completionDetails.CompletionType == CompletionType.ParameterName))
+            switch (completionDetails.CompletionType)
             {
-                // Look for type encoded in the tooltip for parameters and variables.
-                // Display PowerShell type names in [] to be consistent with PowerShell syntax
-                // and now the debugger displays type names.
-                var matches = Regex.Matches(completionDetails.ToolTipText, @"^(\[.+\])");
-                if ((matches.Count > 0) && (matches[0].Groups.Count > 1))
-                {
-                    detailString = matches[0].Groups[1].Value;
-                }
-            }
-            else if ((completionDetails.CompletionType == CompletionType.Method) ||
-                     (completionDetails.CompletionType == CompletionType.Property))
-            {
-                // We have a raw signature for .NET members, heck let's display it.  It's
-                // better than nothing.
-                documentationString = completionDetails.ToolTipText;
-            }
-            else if (completionDetails.CompletionType == CompletionType.Command)
-            {
-                // For Commands, let's extract the resolved command or the path for an exe
-                // from the ToolTipText - if there is any ToolTipText.
-                if (completionDetails.ToolTipText != null)
-                {
-                    // Fix for #240 - notepad++.exe in tooltip text caused regex parser to throw.
-                    string escapedToolTipText = Regex.Escape(completionDetails.ToolTipText);
-
-                    // Don't display ToolTipText if it is the same as the ListItemText.
-                    // Reject command syntax ToolTipText - it's too much to display as a detailString.
-                    if (!completionDetails.ListItemText.Equals(
-                            completionDetails.ToolTipText,
-                            StringComparison.OrdinalIgnoreCase) &&
-                        !Regex.IsMatch(completionDetails.ToolTipText,
-                            @"^\s*" + escapedToolTipText + @"\s+\["))
+                case CompletionType.Variable:
+                case CompletionType.ParameterValue:
+                case CompletionType.Method:
+                case CompletionType.Property:
+                    detailString = completionDetails.ToolTipText;
+                    break;
+                case CompletionType.ParameterName:
+                    // Look for type encoded in the tooltip for parameters and variables.
+                    // Display PowerShell type names in [] to be consistent with PowerShell syntax
+                    // and now the debugger displays type names.
```

Comment:
I think I typo'd that `now`, `how` makes more sense in this comment.

Reply:
fixed!